fenix/app/src/main/java/org/mozilla/fenix/settings/CustomizationFragment.kt

```diff
@@ -140,7 +151,7 @@ class CustomizationFragment : PreferenceFragmentCompat() {
         addToRadioGroup(topPreference, bottomPreference)
     }
 
-    private fun setupGesturesCategory() {
+    private fun setupGesturesCategory(isSwipeToolbarToSwitchTabsVisible: Boolean) {
```

Comment:
I initially missed this new boolean. maybe add a comment that the swipe gesture should not be enabled when the tab strip is visible?